docs/integrations/gitlab-server.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: "docs"
+title: Using Probo with GitLab Server
+class: documentation
+permalink: /integrations/gitlab-server/
+redirect_from: /git/gitlab-server/
+published: true
+---
+You can use Probo.CI with your instance of [GitLab Server](https://about.gitlab.com/install/). Just like the [GitLab Integration](/git/gitlab/ "GitLab Integration"), Probo uses the build status pop-up to display the list of build steps as defined in your `.probo.yaml` file and their statuses. The link can be found in the Overview section of the Merge Request details page.
+
+Follow these steps to get started using Probo with your instance of GitLab Server:
+
+1. <a href="#create-an-oauth-application-on-your-gitlab-server">Create an OAuth application on your GitLab Server.</a>
```

Comment:
Can you use the Markdown syntax for these links? E.g. `[Create an OAuth application on your GitLab Server.](#create-an-oauth-application-on-your-gitlab-server)`